libcxx/test/std/numerics/c.math/hermite.pass.cpp

```diff
@@ -238,78 +196,117 @@ std::vector<T> get_roots(unsigned n) {
             T(5.3874808900112)};
 
   default:
-    throw "Roots of Hermite polynomial of order " + std::to_string(n) + " not implemented!\n";
+    assert(0);
```

Comment:
```suggestion
    assert(false);
```

Reply:
Fixed in 293dd2ba1c8a9f88aadddfb36936ba7d6f01ddc2.